include/vector_io.h

```diff
@@ -12,8 +12,9 @@ namespace quda
   class VectorIO
   {
     const std::string filename;
+#ifdef HAVE_QIO
```

Comment:
Not blocking: Why was this an issue? Also, after looking at `lib/vector_io.cpp`, won't this now throw some warnings related to `parity_inflate` being unused?

Reply:
I think the original issue was that having it outside the ifdef threw those unused errors. So that is why I believe I moved it into the #ifdef QIO since that seemed to be the only time it was used.